other-docs/guides/automatic-updates.md

```diff
@@ -17,12 +17,11 @@ updates:
   - package-ecosystem: "composer"
     # Look for `composer.json` and `composer.lock` files in the `root` directory
     directory: "/"
-    # Create pull requests as soon as updates are made available
-    schedule:
```

Comment:
is this `schedule` line intentionally removed?

Reply:
Nope, not sure how I did that 😅

Reply:
Fixed!